src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -61,7 +61,6 @@ public void processInsolvency(String contextId, String companyNumber,
                         deltaAtFromDbStr)) {
                     insolvencyDocument.setDeltaAt(dateFromBodyRequest);
```

Comment:
Both sides of this `if` block look to be almost exactly the same, apart from the log messages. Do you think we should tidy this up while we are here? It's not a blocker if you would prefer to leave it